compiler_opt/rl/corpus_test.py

```diff
@@ -25,19 +26,21 @@ class CommandParsingTest(tf.test.TestCase):
 
   def test_thinlto_file(self):
     data = ['-cc1', '-foo', '-bar=baz']
-    argfile = self.create_tempfile(content='\0'.join(data))
+    tempdir = self.create_tempdir()
```

Comment:
just curious, why did we need a directory now? just opportunistic improvement? (sgtm, in that case, asking to learn)

Reply:
ah, more of a I forgot to remove after changing the signature of _load_and_parse_command, will revert